pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -87,12 +87,12 @@ type cniBandwidthEntry struct {
 	// IngressRate is the bandwidth rate in bits per second for traffic through container. 0 for no limit. If ingressRate is set, ingressBurst must also be set
 	IngressRate int `json:"ingressRate,omitempty"`
 	// IngressBurst is the bandwidth burst in bits for traffic through container. 0 for no limit. If ingressBurst is set, ingressRate must also be set
```

Comment:
Nit: Can we capitalize `ingressBurst` and `ingressRate` here?

Same nit for `egressRate` and `egressBurst`.

Reply:
Update, thanks for the hint!